src/main/java/com/chessgrinder/chessgrinder/repositories/ClubRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.chessgrinder.chessgrinder.repositories;
+
+import com.chessgrinder.chessgrinder.entities.ClubEntity;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface ClubRepository extends CrudRepository<ClubEntity, UUID> {
+
+    @Override
+    List<ClubEntity> findAll();
```

Comment:
there is another base interface which already returns list from find all
see other repositories